auth.js

```diff
@@ -0,0 +1,39 @@
+const jwtSecret =
+  "your_jwt_secret"; /* This has to be the same key
+used in JWTStrategy(passport.js) */
```

Comment:
No go to expose sensitive data. Let us use a [`.env`](https://www.npmjs.com/package/dotenv) file which we can add to the `.gitignore` file.